tensorbay/cli/auth.py

```diff
@@ -132,17 +136,49 @@ def _update_profile(config_parser: ConfigParser, profile_name: str, arg1: str, a
     click.echo("".join(messages))
 
 
-def _check_args_and_options(arg1: str, arg2: str, get: bool, unset: bool, is_all: bool) -> None:
-    if is_all and not (get or unset):
-        error('Use "--all" option with "--get" or "--unset" option')
+def _check_args_and_options(  # pylint: disable=too-many-arguments
+    arg1: str, arg2: str, get: bool, status: bool, unset: bool, is_all: bool
+) -> None:
+    if is_all and not (get or unset or status):
+        error('Use "--all" option with "--get", "--unset" or "--status" option')
 
-    if get and unset:
-        error('Use either "--get" or "--unset"')
+    if get + unset + status > 1:
+        error('Use at most one of "--get", "--unset" and "--status"')
 
-    if (get or unset) and (arg1 or arg2):
+    if (get or unset or status) and (arg1 or arg2):
         error("Option requires 0 arguments")
 
 
 def _echo_formatted_profile(name: str, value: str) -> None:
     formatted_value = indent(value, INDENT)
     click.echo(f"{name} = {formatted_value}\n")
+
+
+def _status_auth(obj: Dict[str, str], config_parser: ConfigParser, is_all: bool) -> None:
+    if is_all:
+        try:
+            profiles = config_parser["profiles"]
+        except KeyError:
+            sys.exit(1)
+        for key, value in profiles.items():
+            _echo_user_info(key, value)
+        return
+
+    profile_name = obj["profile_name"]
+    try:
+        profile_content = config_parser["profiles"][profile_name]
```

Comment:
name `config_parser["profiles"]` as `profiles`,
should `config_parser["profiles"][profile_name]` name as `profile`?